extras/ams_models/emotions_recognition_model.json

```diff
@@ -0,0 +1,25 @@
+{
+    "model_name": "emotions-recognition-retail-0003",
+    "inputs": [
+        {
+          "input_name": "input",
+          "input_format": "NCHW",
+          "color_format": "BGR",
+          "target_height": 64,
+          "target_width": 64
+        }
+      ],
+    "outputs":
+     [
+            {
+                "output_name": "prob_emotion",
+                "classes": {
+                    "neutral": 0.0,
+                    "happy": 1.0,
+                    "sad": 2.0,
+                    "surprise": 3.0,
+                    "anger": 4.0
+                }
+            }
```

Comment:
Don't we also need value_index mapping here?

Reply:
We do not have any complex mapping here. Just output type and indexed class names. So I do not see the need to index mapping. Maybe in more complex classification classes ? Info in config json file should be enough.